templates/policies/privacy.hbs

```diff
@@ -0,0 +1,203 @@
+      we do with personal information.
+    </p>
+
+    <header>
+      <h2 id="rust-www">rust-lang.org</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      The Rust website at <a href="{{baseurl}}/">rust-lang.org</a> is a project
+      of the <a href="{{baseurl}}/governance/teams/core">Core team</a> and the
+      <a href="{{baseurl}}/governance/teams/community">Community team.</a>
+    </p>
+
+    <dl>
+      <dt>Visitor logs:</dt>
+      <dd>
+        When you visit rust-lang.org, we receive your IP address as part of
+        our standard server logs. We store these logs for 1 year.
+      </dd>
+    </dl>
+
+    <header>
+      <h2 id="crates-io">crates.io</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      <a href="https://crates.io">Crates.io</a> is managed by members of the
+      <a href="{{baseurl}}/governance/teams/crates-io">Crates.io</a> and
+      <a href="{{baseurl}}/governance/teams/core">Rust core</a> teams.
+    </p>
+
+    <dl>
+      <dt>Logging in with GitHub:</dt>
+      <dd>
+        <p>
+          Crates.io requires users to have a
+          <a href="https://www.mozilla.org/firefox/accounts/">GitHub account</a>
+          in order to log in and use the service. When you log-in to Crates.io
+          using a GitHub account, we receive your GitHub username and avatar. If
+          you share a display name or public email address in your GitHub public
+          profile, we also receive that information.
+        </p>
+      </dd>
+
+      <dt>Email addresses:</dt>
+      <dd>
+        <p>
+          You must have a verified email address to publish a crate. We receive
+          any public email address associated with your GitHub account. You can
+          also choose to submit a different address to associate with your
+          Crates.io activity. We will only use your email address to contact you
+          about your account.
+        </p>
+      </dd>
+
+      <dt>Visitor logs:</dt>
+      <dd>
+        <p>
+          When you visit Crates.io, we receive your IP address and user-agent
+          header as part of our standard server logs. We store these logs for 1
+          year.
+        </p>
+      </dd>
+
+      <dt>Information uploaded to crates:</dt>
+      <dd>
+        <p>
+          All crates on Crates.io are public, including the list of crate owners’
+          user names and its upload date. Anyone may view or download a crates’
+          contents. Because of the public nature of Crates.io any personal data
+          you might include in a Cargo.toml file uploaded to a crate will be
+          publicly available. For example, if an email address is in the author’s
+          field in the Cargo.toml file, that email address will also be public.
+        </p>
+        <p>
+          Due to its public nature, be aware if you include any private
+          information in a crate that information may be indexed by search
```

Comment:
I think this needs a comma after "if you include any private information in a crate, that information..."